docs/developer-onboarding.md

```diff
@@ -14,3 +20,43 @@ PaaS is built on CloudFoundary and we use the CloudFoundary CLI to interact with
 
 ### Getting familiar with PaaS
 See [PaaS cheatsheet](/docs/paas-cheatsheet.md).
+
+### Azure CIP
+
+You will need a CIP account in order to raise a PIM (Privileged Identity Management) request to change environment variables or access nightly database backups. Steps on how to register for a CIP account and raise a PIM request
+can be found in the [DfE Technical Guidance](https://technical-guidance.education.gov.uk/infrastructure/hosting/azure-cip/).
+
+## Access to QA, Sandbox and Production
+
+- Navigate to the Support Sign in page: [QA](https://qa.apply-for-teacher-training.service.gov.uk/support/sign-in), [Staging](https://staging.apply-for-teacher-training.service.gov.uk/support/sign-in) and [production](https://www.apply-for-teacher-training.service.gov.uk/support/sign-in)
+- If you already have a DfE Sign-in account, log in. If you do not click, `Create account` and follow the instructions
+- Once you have successfully logged in, you should see a page stating your DfE Sign-in `UUID`
+- Ask a colleague who has support access for the environment you are trying to access to go to the [Add support user page](https://www.apply-for-teacher-training.service.gov.uk/support/users/support/new) and add your email address and your `UUID` (from the last step).
+
+Note: You can also navigate to the Add Support User page by logging into support and clicking `Settings`, `Support users` and `Add support user`
+
+# Additional reading
+
+Have a read through [Teacher Services technical documentation](https://teacher-services-tech-docs.london.cloudapps.digital/#teacher-services-technical-documentation) and [DfE Technical Guidance](https://technical-guidance.education.gov.uk/).
+
+# Developer Slack Channels
```

Comment:
How's this @JR-G? Any thoughts or improvements?

I know you added `#twd_bat_devops`, but I removed it as it just looked like spam in there. I replaced it with `#teacher-services-infra`

Also, sorry, I accidently pushed without pulling your commit down 😓

Reply:
no worries all good – I wasn't aware of that channel myself so looks like even I've gained from the onboarding doc!

Reply:
ha - me too!